plugins/services/src/js/components/forms/GeneralServiceFormSection.js

```diff
@@ -126,23 +124,43 @@ class GeneralServiceFormSection extends Component {
     }
 
     return (
-      <p>
-        {'Need to run a service with multiple containers? '}
-        <a className="clickable" onClick={this.handleOpenConvertToPodModal}>
-          Add another container
-        </a>.
-      </p>
+      <div className="pod pod-narrow pod-short">
```

Comment:
I think it's fine to add margin to the top of this `Need to run a service...` copy, for now. However I think it should only be on the top, so your classes would look like this:
`pod pod-short flush-horizontal flush-bottom`.

In other parts of the form, we're relying on the top margin of heading tags to give separation to the different sections. In the future, I think we should create a `FormSection` component (or something similar) to provide consistent margins, but definitely not part of this PR!